Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1897 @@ label mas_wrs_twitch:
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_15ai:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "A text to speech device, huh? I wonder if my voice will ever be used here...",
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_15ai')
+    return           
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_mobygames",
+            category=['mobygames'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_mobygames:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "What game are you looking for here, [player]?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_mobygames')
+    return    
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_adobepro",
+            category=['premierepro'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_adobepro:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Making a video, [player]?",
+           "Editing footage sure is tough, huh?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_adobepro')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_photoshop",
+            category=['photoshop'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_photoshop:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Doing some image editing, I see."
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_photoshop')
+    return    
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_elyssasite",
+            category=['%%%%%%%%%%%'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_elyssasite:
```

Comment:
If this is in reference to the ddlc easter eggs, we can't really and don't comment on that.